data_safe_haven/infrastructure/programs/sre/user_services.py

```diff
@@ -93,30 +95,36 @@ def __init__(
         child_opts = ResourceOptions.merge(opts, ResourceOptions(parent=self))
         child_tags = tags if tags else {}
 
-        # Deploy the Gitea server
-        SREGiteaServerComponent(
-            "sre_gitea_server",
-            stack_name,
-            SREGiteaServerProps(
-                containers_subnet_id=props.subnet_containers_id,
-                database_subnet_id=props.subnet_containers_support_id,
-                database_password=props.gitea_database_password,
-                dns_server_ip=props.dns_server_ip,
-                dockerhub_credentials=props.dockerhub_credentials,
-                ldap_server_hostname=props.ldap_server_hostname,
-                ldap_server_port=props.ldap_server_port,
-                ldap_username_attribute=props.ldap_username_attribute,
-                ldap_user_filter=props.ldap_user_filter,
-                ldap_user_search_base=props.ldap_user_search_base,
-                location=props.location,
-                resource_group_name=props.resource_group_name,
-                sre_fqdn=props.sre_fqdn,
-                storage_account_key=props.storage_account_key,
-                storage_account_name=props.storage_account_name,
-            ),
-            opts=child_opts,
-            tags=child_tags,
-        )
+        # Deploy the Gitea servers
+        if props.gitea_servers == GiteaServers.BOTH:
+            gitea_servers = ["external", "internal"]
+        else:
+            gitea_servers = ["internal"]
```

Comment:
```suggestion
        gitea_servers = ["external", "internal"] if (props.gitea_servers == GiteaServers.BOTH) else ["internal"]
```
not sure this is clearer though

Reply:
I'm also switching to using a boolean True/False for deploying the external mirror anyway, as we'd (presumably) always want the internal one anyway

Reply:
Sounds like I should stop commenting on your WIP PR 😆

Reply:
Already some good points to nip some things in the bud 😆